packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio-group/gux-form-field-radio-group.tsx

```diff
@@ -90,6 +90,12 @@ export class GuxFormFieldRadioGroupBeta {
     if (radioSlots) {
       radioSlots.forEach(item => {
         item.hasGroupError = hasGroupError;
+        const radioInput = item.querySelector('input');
+        if (hasGroupError) {
+          radioInput.setAttribute('aria-invalid', 'true');
+        } else {
+          radioInput.removeAttribute('aria-invalid');
+        }
```

Comment:
Linked to previous comments: whatever you decide for checkbox group do here too.